src/pages/settings/Security/CloseAccountPage.js

```diff
@@ -76,7 +76,7 @@ class CloseAccountPage extends Component {
                             value={this.state.reasonForLeaving}
                             onChangeText={reasonForLeaving => this.setState({reasonForLeaving})}
                             label={this.props.translate('closeAccountPage.enterMessageHere')}
-                            containerStyles={[styles.mt5]}
+                            containerStyles={[styles.mt5, styles.closeAccountMessage]}
```

Comment:
On second thought, maybe we should name this `closeAccountMessageInput` so it's a bit more descriptive.

Reply:
Done.